rkt/image/dockerfetcher.go

```diff
@@ -97,7 +97,7 @@ func (f *dockerFetcher) fetch(u *url.URL) (*os.File, error) {
 
 	registryURL := strings.TrimPrefix(u.String(), "docker://")
 	user, password := f.getCreds(registryURL)
-	acis, err := docker2aci.Convert(registryURL, true /* squash */, tmpDir, tmpDir, user, password, f.InsecureFlags.AllowHTTP())
+	acis, err := docker2aci.Convert(registryURL, true /* squash */, tmpDir, tmpDir, d2acommon.GzipCompression, user, password, f.InsecureFlags.AllowHTTP())
```

Comment:
Since this is just a temporary ACI and we're gonna import it in the CAS (which stores uncompressed ACIs) I'd use NoCompression.